src/test/java/org/broadinstitute/consent/http/resources/UserResourceTest.java

```diff
@@ -373,45 +380,66 @@ public void testGetUsersByInstitutionSuccess() {
   }
 
   @Test
-  public void testUpdate() {
+  public void testUpdateSelf() {
```

Comment:
Some of the changes shown here are just appearing because I moved `testUpdate` down with the other tests for `userResource.update` instead of by the `userResource.updateSelf` tests. Actual change is modifying tests to make sure `supportRequestService.handleSuggestedUserFieldsSupportRequest` is called the appropriate number of times, and that we get an error status if it fails